### PR TITLE
Add a CONTRIBUTING file. r=azasypkin

### PR DESCRIPTION
**Pull request by gmarty**
This is a first stab at a page about contribution.
@azasypkin: What do you think?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+[list of good first bugs](https://github.com/fxbox/app/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+## Coding standard
+
+The code base is written in es2015, make sure to use semantic and syntactic
+idioms in this flavour of JavaScript.
+
+The coding standard is not documented but most of the important features are
+enforced using eslint.
+To ensure your code is compatible a pre-commit hook run. Alternatively, you can
```
Comment (azasypkin) [inline CONTRIBUTING.md]: __nit:__ `hook is run`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Unlike coding standard, the coding style is not yet enforced or documented.
+Please be familiar with the code base and make sure you respect the style in
+place.
+
+As an example of coding style, the promises are usually split and indented this
+way:
+```javascript
+api.doPpromise()
+  .then(result => {
```
Comment (azasypkin) [inline CONTRIBUTING.md]: __nit:__ `.then((result) => {` :)

**Comment by azasypkin** [comment]
Looks good! r+ with few tiny nits handled.

File: CONTRIBUTING.md (hunk 2 above)
Comment (gmarty) [inline CONTRIBUTING.md]: Ah ah! Thanks for noticing.

**Pull request merged**